lib/agent/providers/network/index.js

```diff
@@ -118,3 +119,16 @@ exp.get_open_access_points_list = function(callback){
   });
 
 };
+
+exp.get_connection_status = function(cb) {
```

Comment:
I would use `control.preyproject.com/healthz` as the health endpoint is assured to respond 99.99% of the times